README.markdown

```diff
@@ -10,220 +10,134 @@ There are two ways in which you can install OneZoom on a personal computer: full
-	* set `is_testing = False` in `models/db.py` and `migrate=0` in appconfig.ini.
 
+For any installation, OneZoom requires node & python (3.11).
 
-## Downloading the OZtree app
+#### Debian/Ubuntu
 
-Download a copy of the OZtree application from GitHub at [https://github.com/OneZoom/OZtree](https://github.com/OneZoom/OZtree), either as a zip file (not recommended), or probably better (easier to update), by cloning the repository (e.g. using `git clone https://github.com/OneZoom/OZtree` or if you have [GitHub Desktop](https://desktop.github.com) installed, click "Open in Desktop" from the [OZtree repo](https://github.com/OneZoom/OZtree)). Make sure the git folder is called "OZtree" (this is the default when you clone the repo, but not if you download it as a zip file).
-
-For full installation, you will also need to download the source code version of web2py, either via git (https://github.com/web2py/web2py/) or simply from the download link at http://www.web2py.com/. You can then place the OZtree directory into the `applications` directory of the web2py folder.
+```
+apt install nodejs npm
+apt install python3 python3-dev python3-venv
+```
 
+### Full installation
 
-## Building the OneZoom tree viewer
+In addition, for a full installation you also need nginx, supervisor & MySQL.
 
-Compiling and creating the OneZoom explorer javascript code requires grunt to be installed. This compiles javascript code from multiple sources into a single file. You will need to install the node package manager, npm, then do 
+#### Debian/Ubuntu
 
 ```
-npm install -g grunt-cli
-```
+apt install nginx
 
-To install this is likely to require administrator privileges. Other required packages can then be installed from within the OZtree folder by simply typing the following (which may take a while to complete!)
+apt install supervisor
 
-```
-npm install
+apt install lsb-release
+wget https://dev.mysql.com/get/mysql-apt-config_0.8.29-1_all.deb
+dpkg -i mysql-apt-config_0.8.29-1_all.deb
+apt update && apt install mysql-server
+# NB: Select "Use Legacy Authentication Method (Retain MySQL 5.x Compatibility)"
 ```
 
-Once these are installed you can run grunt as follows (feel free to examine the configuration options which are stored in `Gruntfile.js` in the main OZtree directory):
+#### Windows
 
-#### Compile documentation
-`grunt docs`: Use this command to generate a compiled documentation file. This will generate a large compiled markdown file in `OZprivate/rawJS/OZTreeModule/docs/_compiled.markdown`, which is best viewed once you have got web2py running, by pointing your browser to `dev/DOCS` (e.g. at `http://127.0.0.1:8000/dev/DOCS`).  Note that viewing this page requires a working internet connection to get various formatting files)
+On Windows we recommended downloading the MSI installer as it will make it easier to configure the new server during the installation
```

Comment:
Clarify that this refers to the MySQL MSI, and ideally link to it.

Also, missing period after installation.